cumulus/zombienet/tests/0008-elastic_authoring.zndsl

```diff
@@ -0,0 +1,15 @@
+Description: Slot based authoring for elastic scaling
+Network: ./0008-elastic_authoring.toml
+Creds: config
+
+alice: is up
+collator-elastic: is up
+collator-single-core: is up
+
+
+# configure relay chain
+alice: js-script ./assign-core.js with "2100,0" return is 0 within 600 seconds
+alice: js-script ./assign-core.js with "2100,1" return is 0 within 600 seconds
+
+collator-single-core: reports block height is at least 20 within 225 seconds
+collator-elastic: reports block height is at least 40 within 225 seconds
```

Comment:
I think this second check should have a close to zero timeout since the checks are sequential (because if the single core had enough time to back and include 20 blocks already, the elastic scaling one should have done twice the blocks in the same amount of time)